pkg/selinux/setroubleshoot-view.jsx

```diff
@@ -254,11 +280,22 @@ var SETroubleshootPage = React.createClass({
             }
             var entries = this.props.entries.map(function(itm) {
                 itm.runFix = self.props.runFix;
+                var listingCaption;
+                if (itm.details && 'firstSeen' in itm.details) {
+                    if (itm.details.reportCount >= 2)
+                        listingCaption = cockpit.format(_("Occurred between $0 and $1"), itm.details.firstSeen, itm.details.lastSeen);
+                    else
+                        listingCaption = cockpit.format(_("Occurred $0"), itm.details.firstSeen);
+                }
+                var onDeleteClick;
+                if (itm.details)
+                    onDeleteClick = self.handleDeleteAlert.bind(self, itm.details.localId);
                 var dismissAction = (
                     <button
                         title="Dismiss"
                         className="pficon pficon-delete btn btn-danger"
-                        disabled />
+                        onClick={onDeleteClick}
+                        disabled={ !onDeleteClick || !('deleteAlert' in self.props && self.props.deleteAlert) } />
```

Comment:
`'deleteAlert' in self.props && self.props.deleteAlert` can be simplified to `self.props.deleteAlert`.